src/lib/templateHelpers.test.ts

```diff
@@ -0,0 +1,197 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { expect } from "chai";
+import {
+  addNamespace,
+  formatForTsDoc,
+  isCommonPathParameter,
+  isCommonQueryParameter,
+} from "./templateHelpers";
+import { commonParameterPositions } from "@commerce-apps/core";
+
+describe("When adding namespaces to individual content (types)", () => {
+  it("Prefixes the namespace successfully ", () => {
+    expect(addNamespace("foo", "types"))
+      .to.be.a("string")
+      .and.equal("types.foo");
+  });
+
+  it("Will not prefix void with a namespace", () => {
+    expect(addNamespace("void", "types")).to.be.a("string").and.equal("void");
+  });
+
+  it("Will not prefix object with a namespace", () => {
+    expect(addNamespace("object", "types"))
+      .to.be.a("string")
+      .and.equal("object");
+  });
+
+  it("Will not prefix Object with a namespace", () => {
+    expect(addNamespace("object", "types"))
+      .to.be.a("string")
+      .and.equal("object");
+  });
+
+  it("Throws an error when called with undfined content", () => {
+    expect(() => addNamespace(null, "types")).to.throw("Invalid content");
+  });
+
+  it("Throws an error when called with null content", () => {
+    expect(() => addNamespace(undefined, "types")).to.throw("Invalid content");
+  });
+
+  it("Throws an error when the type is not a valid string", () => {
```

Comment:
add few more tests for empty string, string with only spaces

Reply:
This was moved wholesale. I would prefer not to much else beyond the refactor to keep it focused.